### PR TITLE
Cypress tests for the apply form

### PR DESCRIPTION
**Pull request by cecastosic**
Screenshot 2020-10-09 at 11 22 19
Screenshot 2020-10-09 at 11 19 34

### CODE REVIEW

**Review by benna100 — CHANGES_REQUESTED** [review]
Please make the PR to develop not master.

Why is the why.docs file empty?

Please add a readme.md with documentation :)

**Comment by benna100** [comment]
But **really** good work Svetlana! Thank you so much for this!!

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+module.exports = (on, config) => {
+  // `on` is used to hook into various events Cypress emits
+  // `config` is the resolved Cypress config
+}
```
[the remark this answers is not in the document]
Comment (cecastosic) [reply cypress/plugins/index.js:21]: Once we decide to write plugins (custom code that executes during particular stages of the Cypress lifecycle) the code should be written here. The both, folder and file come with cypress installation.

File: cypress/support/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
[the remark this answers is not in the document]
Comment (cecastosic) [reply cypress/support/index.js:20]: By default Cypress will automatically include the support file cypress/support/index.js. This file runs before every single spec file. We do this purely as a convenience mechanism so you don’t have to import this file in every single one of your spec files. https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Support-file

File: cypress/support/commands.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+import 'cypress-file-upload'
```
[the remark this answers is not in the document]
Comment (cecastosic) [reply cypress/support/commands.js:26]: https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-files

**Comment by cecastosic** [comment]
> Please make the PR to develop not master.
> 
> Why is the why.docs file empty?
> 
> Please add a readme.md with documentation :)

Done
why.doc is not empty 
Screenshot 2020-10-25 at 00 07 15

File: cypress/plugins/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (benna100) [reply cypress/plugins/index.js:21]: Okay, cool!

File: cypress/support/commands.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (benna100) [reply cypress/support/commands.js:26]: Okay, fair :)

File: cypress/support/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (benna100) [reply cypress/support/index.js:20]: Okay, cool!

**Review by benna100 — CHANGES_REQUESTED** [review]
The documentation that you have in the why.docs what is that?

Documentation for cypress should be written in a readme.md file. Maybe you can add a testing section to this readme: https://github.com/HackYourFuture-CPH/hackyourfuture.dk/blob/master/README.md

**Comment by cecastosic** [comment]
> The documentation that you have in the why.docs what is that?
> 
> Documentation for cypress should be written in a readme.md file. Maybe you can add a testing section to this readme: https://github.com/HackYourFuture-CPH/hackyourfuture.dk/blob/master/README.md

I just put questions from the form in the why.doc file to not be empty.
Regarding readme, I added readme here https://github.com/cecastosic/hackyourfuture.dk/blob/test/cypress/README.md

**Comment by benna100** [comment]
Ahh perfect, thanks :)

**Review by benna100 — APPROVED** [review]

**Pull request merged**